app/assets/stylesheets/darkswarm/map.css.scss

```diff
@@ -63,3 +63,8 @@
     left: 0px;
   }
 }
+
+.tabs-content .map-footer {
+  position: relative;
```

Comment:
just out of curiosity why do we need to position it relatively? I don't see any absolute counterpart. Perhaps a screenshot will be best to explain it.

Reply:
It's overlaid on the map, but in this case it's in the group page inside a tab. On the main map page the positioning is very different and fills the entire screen.